packages/shared/components/calculator-level/calculator-slider/calculator-slider.tsx

```diff
@@ -0,0 +1,91 @@
+import "./calculator-slider.scss";
+
+import Slider from "rc-slider";
+import * as React from "react";
+import NumberFormat from "react-number-format";
+
+class _CalculatorSlider extends React.PureComponent<Props, State> {
+  state = {
+    value: this.props.defaultValue
+  };
+
+  static defaultProps = {
```

Comment:
Don't use defaultProps.
Use `const { min, max, title, step = 1 , valueSuffix } = this.props;` instead it.

Reply:
Done